notebooks/data/DownloadLondonPostcodeZones.py

```diff
@@ -0,0 +1,43 @@
+# Databricks notebook source
```

Comment:
I would love to move all the data download code in the main notebook, this makes it easier to get started as you only need to run a single notebook.
If we add a SQL notebook later, we can duplicate the python code in a first cell of the SQL notebook as well. It's not that bad to duplicate the download code if it makes it easier to get started IMHO.

Reply:
I wouldnt, IMHO it makes the example more complicated by a code we have spent less time on (since it is just a download). Makes the examples less clean with more distractions.
The message is basically:
If you dont have the data please use the download notebook. 
If you have the data use the data you have, point to those tables.

Also it is the way we do for other notebooks, so I wouldnt delay this PR over that.
We can standardise later for this if we all agree it is the way we want to do examples - ie. all in a single notebook

Reply:
Cool, I agree